.clang-format

```diff
@@ -0,0 +1,123 @@
+---
+# General options
+Language: Cpp
+Standard: c++17
+DisableFormat: false
+
+AccessModifierOffset: -4
+AlignAfterOpenBracket: AlwaysBreak
+AlignConsecutiveAssignments: false
+AlignConsecutiveBitFields: false
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: false
+AlignEscapedNewlines: Right
+AlignOperands: DontAlign
```

Comment:
Somehow I get an error when I try to use this style file with clang-format-11 or clang-format-10

```
YAML:14:16: error: invalid boolean
AlignOperands: DontAlign
               ^~~~~~~~~
```